src/connectdlg.cpp

```diff
@@ -80,6 +86,11 @@ CConnectDlg::CConnectDlg ( CClientSettings* pNSetP, const bool bNewShowCompleteR
                                "<br>" + tr ( "Permanent servers (those that have been listed for longer than 48 hours) are shown in bold." ) +
                                "<br>" + tr ( "You can add custom directories in Advanced Settings." ) );
 
+    lvwServers->setToolTip ( "<b>" + tr ( "Server List" ) + ":</b> " +
+                             tr ( "The list of servers registered with the selected directory. "
```

Comment:
Maybe even shorter: Just mention the double click to join a server. The other option doesn't really need to be explained and is slower.

Reply:
Server list: Double click a Server to connect.